src/DisplayBoard.cpp

```diff
@@ -0,0 +1,110 @@
+#include "DisplayBoard.hpp"
+
+namespace
+{
+    using u_int = unsigned int;
+
+    const char HORIZONTAL = '-';
+    const char VERTICAL = '|';
+    const char INTERSECTION = '+';
+    const char PADDING = ' ';
+
+
+    std::string drawPadding(const u_int width, const char sign = PADDING);
+    std::string drawEndLine(const u_int width);
+    std::string drawEmptyLine(const u_int width, const char sign = PADDING);
+    std::string drawColumns(const u_int maxElementsInRows, const u_int colNumber, const u_int heightCol);
+    std::string drawRow(const u_int maxElementsInRows);
+    std::stringstream drawBoard(const u_int width,
+                                const u_int height,
+                                const u_int widthRows,
+                                const u_int heightCol);
+}
+
+namespace DisplayBoard
+{
+    std::string displayInterface(const Board& b){
+        auto width = b.getRowsNumber();
+        auto height = b.getColsNumber();
+        auto widthRows = b.findLongestVectorInRows();
+        auto heightCol = b.findLongestVectorInCols();
+
+        auto tmp = drawBoard(width, height, widthRows, heightCol);
+        auto tmpStr = tmp.str();
+        std::cout << tmpStr << "\n";
+
+        return tmpStr;
```

Comment:
Function should have return stringstream or string only. Printing drawBoard on cout in meantime makes no other sense then for testing purposes.

Reply:
Ok, changed.

Reply:
Imo, this is helper method and it should not be dependent in any way from Object. It should only take params as was described in assumptions of 2.2.
@LordLukin what do you think